src/Akka.Streams.Kafka/Messages/CommittableOffsetBatch.cs

```diff
@@ -17,7 +18,7 @@ internal sealed class CommittableOffsetBatch : ICommittableOffsetBatch
         /// CommittableOffsetBatch
         /// </summary>
         public CommittableOffsetBatch(IImmutableDictionary<GroupTopicPartition, OffsetAndMetadata> offsetsAndMetadata, 
-                                      IImmutableDictionary<string, IInternalCommitter> committers, 
+                                      IImmutableDictionary<GroupTopicPartition, KafkaAsyncConsumerCommitter> committers, 
```

Comment:
We had a blend of `string`s and other stuff flying around. Re-organized this on `GroupTopicPartition` for `committers` since those _do_ need to incorporate `groupId` and `Partition` in addition to topic.